development/.env

```
@@ -0,0 +1,4 @@
+TELEGRAM_TOKEN=ABCDE <- Your TOKEN
+VK_TOKEN=ABCDE
+BOT_NAME=Nikita
```

I think we should remove the right side and write about the filling and links at Readme

```suggestion
TELEGRAM_TOKEN=
VK_TOKEN=
BOT_NAME=
```